MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -5288,8 +5290,10 @@
 			files = (
 			);
 			inputPaths = (
+				"~/aadtests/conf.json",
```

Comment:
Do these have to be here?

Reply:
Yes. The build script will be run only if input or output files will change.